src/app/models/user.model.ts

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright 2020 Google LLC
+ */
+export interface User {
+  uid: string | undefined;
```

Comment:
Can we not just say `uid?: string` here?

Reply:
Yep, good point. (Still new to the ? operator so I missed it)